.travis.yml

```diff
@@ -1,7 +1,16 @@
 dist: trusty
+
 language: java
+
 jdk:
-  - oraclejdk9
+  - oraclejdk11
+
 notifications:
   email:
-      - notification@qos.ch
+      - notification@qos.c
```

Comment:
strange, not ?

Reply:
Good catch. That's 5ddfa9c6b3252834cf3be5f88ea27c821f70a577 which is now on master. @ceki you want to fix that

Reply:
@twillouer  thank you. Fixed in the 2.0 branch